packages/sdk/src/oracle/services/optimisticOracleV2.ts

```diff
@@ -24,10 +25,10 @@ export class OptimisticOracleV2 implements OracleInterface {
   constructor(protected provider: Provider, protected address: string, public readonly chainId: number) {
     this.contract = connect(address, provider);
   }
-  private upsertRequest = (request: Omit<Request, "chainId">): Request => {
+  private upsertRequest = (request: RawRequest): Request => {
```

Comment:
this funcction now has the additional purpose of converting a raw request ( straight from contract) to our normalized request ( common interface) for the ui to consume

Reply:
This is nice +1